backend/util/importRecipes.js

```diff
@@ -37,38 +49,42 @@ module.exports = function(recipeModel, recipeIngredientIndexModel, recipeStepMod
             ImgLink: img,
             Prep: prepTime,
             CookTime: cookingTime,
-            Servings: servings
-        })
+            Servings: servings,
+            Calories: calories,
+            Fat: fat,
+            Carbs: carbs,
+            Protein: protein,
+        });
 
         recipe.steps.forEach((step) => {
             bulkRecipeSteps.push({
                 StepID: uuidv4(),
                 OrderOfStep: Number(step.order),
                 StepInfo: step.description.trim(),
-                RecipeID: pk
-            })
-        })
-        
+                RecipeID: pk,
+            });
+        });
+
         // Associate this recipe with the ingredient for DISPLAY
-        const rawIngredientStr = recipe.ingredients_raw
+        const rawIngredientStr = recipe.ingredients_raw;
         rawIngredientStr.forEach((str) => {
             bulkRecipeIngredientStr.push({
                 RecipeID: pk,
                 RawIngredient: str,
-                RecipeIngredientID: uuidv4()
-            })
-        })
+                RecipeIngredientID: uuidv4(),
+            });
+        });
 
         // Associate this recipe with the ingredients for SEARCH
         const ingredients = recipe.ingredients_parsed;
         ingredients.forEach((ingredient) => {
             const ingredientPK = getPrimaryKey(ingredient);
             bulkRecipeIngredientIndex.push({
                 IngredientID: ingredientPK,
-                RecipeID: pk
-            })
-        })
-    })
+                RecipeID: pk,
+            });
+        });
+    });
 
     // recipeModel.bulkCreate(bulkRecipes, {
```

Comment:
can we double check with @nigel5 whether we need this commented code here?

Reply:
Had that commented out because I was running other imports. For the first time run all the bulk INSERTS should not be commented out

Reply:
Uncommented for this PR